### PR TITLE
Update sample app to latest tools/SDK

### PR DESCRIPTION
**Pull request by violinyanev**
Fixes all issues reported by Android Studio, uses modern libs and classes, and complies better to Android best practices.

Gradle: 7.4.2
CompileSDK: 33
Uses view binding instead of kotlin synthetics
New Gradle DSL syntax
Exports activity
Rename app name to not contain "Kotlin", that's obvious

### CODE REVIEW

**Review by bojackHaasman — CHANGES_REQUESTED** [review]
I can build and run the app successfully, but the camera properies need to be adjusted
grafik

**Comment by violinyanev** [comment]
> I can build and run the app successfully, but the camera properies need to be adjusted grafik

It's not related to current PR, fixed in a separate one: https://github.com/bmwcarit/ramses-sample-app/pull/14

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
         android:roundIcon="@mipmap/ramseslogo_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".MainActivity" android:exported="true">
```
Comment (bojackHaasman) [inline app/src/main/AndroidManifest.xml]: I don't think we should need this:
grafik

File: gradle.properties [hunk 2]
```diff
@@ -1,11 +1,3 @@
-//  -------------------------------------------------------------------------
-//  Copyright (C) 2021 BMW AG
-//  -------------------------------------------------------------------------
-//  This Source Code Form is subject to the terms of the Mozilla Public
-//  License, v. 2.0. If a copy of the MPL was not distributed with this
-//  file, You can obtain one at https://mozilla.org/MPL/2.0/.
-//  -------------------------------------------------------------------------
-
```
Comment (bojackHaasman) [inline gradle.properties]: why?

**Review by bojackHaasman — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 2 above)
Comment (violinyanev) [reply gradle.properties]: Android Studio doesn't like the header and reports syntax issues. It's a configuration file, so it doesn't strictly need the header

Comment (bojackHaasman) [reply gradle.properties]: yes, because a comment in this file starts with "#"

**Review by bojackHaasman — CHANGES_REQUESTED** [review]

**Review by bojackHaasman — APPROVED** [review]

**Review by smirko-dev — APPROVED** [review]

**Pull request merged**